test/test_transformer.cc

```diff
@@ -518,3 +518,211 @@ TEST(Transformer, Codegen) {
   ASSERT_TRUE(loop4pos > loop3pos);
   ASSERT_TRUE(stmtpos > loop4pos);
 }
+
+static isl::schedule_node tile(isl::schedule_node node,
+			       std::vector<int> tileSizes,
```

Comment:
Please do _not_ mix tabs and spaces.  Generally, run clang-format with LLVM style on your code.

Reply:
Yeah sorry, I forgot to run clang-format.

Reply:
You can make clang-format into a git precommit hook, either to auto-format your commit or to reject it so that you can format it.